conode/Makefile

```diff
@@ -29,20 +33,22 @@ newdb: conode
 	echo; echo "Shutting down conodes"; pkill -x conode; rm -f conodes/running
 
 # Use this target to build from local source
-docker: build/Dockerfile conode
+docker: build/Dockerfile cothority-pull
 	@export GO111MODULE=on GOOS=linux GOARCH=amd64; \
-	go build -o build/conode; \
-	cd ../cothority && \
-		go build -o ../conode/build/bcadmin ./byzcoin/bcadmin && \
-		go build -o ../conode/build/csadmin ./calypso/csadmin
+	go build -o build/conode
```

Comment:
You can extract this build.

Reply:
no, because once its for the target machine, once it's for the Dockerfile, which, on my computer, won't like the mac-binary.